packages/bpk-component-image/README.md

```diff
@@ -111,18 +111,19 @@ export default () => (
 
 ## Props
 
-| Property                     | PropType  | Required | Default Value       |
-| ---------------------------- | --------- | -------- | ------------------- |
-| altText                      | string    | true     | -                   |
-| height                       | number    | true     | -                   |
-| src                          | string    | true     | -                   |
-| width                        | number    | true     | -                   |
-| className                    | string    | false    | null                |
-| inView                       | bool      | false    | true                |
-| loading                      | bool      | false    | false               |
-| onLoad                       | func      | false    | null                |
-| style                        | object    | false    | {}                  |
-| suppressHydrationWarnings    | bool      | false    | false               |
+| Property                     | PropType                         | Required | Default Value             |
+| ---------------------------- | -------------------------------- | -------- | ------------------------- |
+| altText                      | string                           | true     | -                         |
+| height                       | number                           | true     | -                         |
+| src                          | string                           | true     | -                         |
+| width                        | number                           | true     | -                         |
+| borderRadiusStyle            | oneOf(BORDER_RADIUS_STYLES)      | false    | BORDER_RADIUS_STYLES.none |
```

Comment:
Sorry to be a pain, but:

```suggestion
| borderRadiusStyle            | oneOf(BORDER_RADIUS_STYLES.none, BORDER_RADIUS_STYLES.sm)      | false    | BORDER_RADIUS_STYLES.none |
```

That's how the [RN card one](https://backpack.github.io/components/card?platform=native) does it so it's consistent then.